packages/cli/CHANGELOG.md

```diff
@@ -5,6 +5,31 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 A list of unreleased changes can be found [here](https://github.com/SAP/ui5-cli/compare/v4.0.26...HEAD).
 
 <a name="v4.0.26"></a>
```

Comment:
The anchors seem to be set incorrectly. Has this been generated by release please or was it a manual edit?